repl/src/main/resources/fake_shell.py

```diff
@@ -523,6 +524,8 @@ def main():
             exec('from pyspark.sql import HiveContext', global_dict)
```

Comment:
I think this is imported for PySpark RSC support.
I think in the long run we should separate repl code and rsc code to 2 separate python files. And we can fix this when we add SparkSession support for RSC.